index.html

```diff
@@ -1412,6 +1412,91 @@ <h3>Biometrics</h3>
+    </p>
+
+    <p>
+      The <a 
+      href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">W3C TAG
+      Ethical Web Principles</a> document is a helpful guide
+      when weighing ethical considerations of various technologies.
+    </p>
+
+    <section class="informative">
+      <h3>Broad Ethical Principles</h3>
+      <p>
+        Decentralized Identifiers may come to underpin much of our digital life. This 
+        may include public social and career personas, as well as more private personas 
+        shared among friends and family. Identifiers representing these personas and 
+        relationships may be some of the most important in our lives, so great care 
+        will be taken when choosing an identifier system to support these critical 
+        activities. As with all things, the appropriate and ethical use of technology
+        will be strongly considered  when implementing items related to DIDs.
+      </p>
+
+      <p>
+        As noted in the <a
+        href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#principles">
+        Principles</a> section of the Ethical Web Principles, there are key goals that
+        apply to all Web standards and technologies. DIDs are explicitly intended to
+        help users get closer to several of these goals, especially the following:
+        <ul>
+          <li>
+            <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">
+              Privacy
+            </a>
+          </li>
+          <li>
+            <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">
+              Individual Control
+            </a>
+          </li>
+          <li>
+            <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">
+              Device Independence
+            </a>
+          </li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Other concerns as applied to DID method development</h3>
+      <p class="advisement">
+        The following section reflects the views of some members of the working group.
+        Additional PRs are welcome from the working group with additional points of view.
+      </p>
+
+      <section class="informative">
+        <h3>Environmental Considerations</h3>
+
+        <p>
+          The guiding principle that 
+          <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#sustainable">the
+          web must be an environmentally sustainable platform</a> should be followed.
+          <br />
+          When implementing or utilizing a DID method, consideration should be given
+          to the environmental impacts caused by the method.
+        </p>
+
+        <p class="issue">
+          Some members of the working group feel it important that the [[DID-RUBRIC]] 
+          include environmental considerations as discussed in <a
+          href="https://github.com/w3c/did-rubric/pull/51">this pull request</a> in
```

Comment:
it's pretty common to cite issues or PRs when work is in progress, as a way on unblocking work. We've done this a few times in did core if you prefer to cite an issue instead of the PR, we can make that change.... for example:

https://github.com/w3c/did-spec-registries/blob/main/index.html#L1061

Reply:
I'd agree if we were earlier in the publication process, but this is now a published note, so its not really a "work in progress" as much as, say, the current VC-API spec.

Seems to me, if we want to point to specifics in the Rubric, we should point to published criteria, and if necessary, keep issues open in the implementation guide repo until we get good criteria added to the Rubric.

Reply:
my point is that its bad form to object to adding a link to an open issue / PR, as a mechanism for resolving a contentious issue... we make progress incrementally, sometimes that acknowledges that issues remain open... For example:

https://github.com/w3ctag/design-principles/issues/338
https://github.com/w3ctag/design-principles/issues/339
https://github.com/w3ctag/design-principles/issues/239

Could all be cited since they are all relevant to the imp guide, and all remain unresolved.

Reply:
@OR13 any issue in pulling the PR reference as this issue paragraph will go away soon enough with just a pointer to the rubric?

Reply:
I don't have any issues merging PRs... git tracks the history of all changes and all comments.

Reply:
@jandrieu -- The published NOTE is a static document, which does not and will not include the reference to the PR on another document. There's nothing wrong with such a reference in an Editor's Draft, on one of which all these current PRs are focused, and from which all such references would be removed before it were published as an updated NOTE.

Reply:
@TallTed As I understand it, echidna now automatically publishes updates to the document. So, all PRs merged get published.

In short, I don't believe this is an editor's draft any longer.

Maybe @iherman can correct me if I'm wrong.